apps-rendering/src/components/WithAgeWarning/index.tsx

```diff
@@ -0,0 +1,185 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import type { Tag } from '@guardian/content-api-models/v1/tag';
+import { ArticleDesign, ArticleDisplay } from '@guardian/libs';
+import type { ArticleFormat } from '@guardian/libs';
+import { from, remSpace } from '@guardian/source-foundations';
+import { AgeWarning } from '@guardian/source-react-components-development-kitchen';
+import type { Option } from '@guardian/types';
+import { OptionKind } from '@guardian/types';
+import { grid } from 'grid/grid';
+import type { Item } from 'item';
+import { isComment, isNews } from 'item';
+import { articleWidthStyles, wideContentWidth } from 'styles';
+
+interface WithAgeWarningProps {
+	item: Item;
```

Comment:
Is it worth passing `tags` and `series` as props rather than the full `Item` type? That should significantly reduce the domain of this component.

Reply:
yes good point, made the change